oops/oops_test.go

```diff
@@ -70,6 +71,25 @@ func doubleWrapf() error {
 	return oops.Wrapf(oops.Wrapf(oops.Wrapf(rootCause, "yuck"), "bad"), "why would you do this")
 }
 
+func stripPathPrefix(s string) (string, error) {
+	// Strip path prefixes from filenames, aids comparisons in stack trace tests
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		return "", errors.New("cannot determine caller filename")
+	}
+	pth := path.Dir(filename)
+	return strings.ReplaceAll(s, pth, "github.com/samsarahq/go/oops"), nil
+
+}
+
+func fixLineNumbers(s string) string {
+	// Standardize line numbers in captured stack traces. These can vary depending on the go version in use at runtime, also
```

Comment:
Just so I'm clear, what is the intended "standardization" `fixLineNumbers` performs? Could before/after examples clarify?

Reply:
Thanks @bharling - that clarifies a lot.